app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -650,6 +662,85 @@ protected void updatePlaybackSpeedButtonText() {
         // Only meaningful on AudioplayerActivity, where it is overridden.
     }
 
+    /**
+     * Abstract directions to skip forward or back (rewind) and encapsulates behavior to get or set preference (including update of UI on the skip buttons).
+     */
+    static public enum SkipDirection {
+        SKIP_FORWARD(UserPreferences::getFastForwardSecs, MediaplayerActivity::getTxtvFFFromActivity, UserPreferences::setFastForwardSecs, R.string.pref_fast_forward),
+        SKIP_REWIND(UserPreferences::getRewindSecs, MediaplayerActivity::getTxtvRevFromActivity,  UserPreferences::setRewindSecs, R.string.pref_rewind);
+
+        private final Supplier<Integer> getPrefSecsFn;
+        private final Func1<MediaplayerActivity, TextView> getTextViewFn;
```

Comment:
Nope, not in this case.  The activity is passed to a static method that does not store the activity or use it in some deferred mechanism that would hold a reference.